pkg/virt-controller/watch/vmi.go

```diff
@@ -638,6 +666,17 @@ func (c *VMIController) updateStatus(vmi *virtv1.VirtualMachineInstance, pod *k8
 	return nil
 }
 
+func prepareAnnotationsPatchAddOp(key, value string) (string, error) {
+	valueBytes, err := json.Marshal(value)
+	if err != nil {
+		return "", fmt.Errorf("failed to prepare new annotation patchOp for key %s: %q", key, err)
```

Comment:
I think `%v` needs to be used here (instead of `%q`).

Reply:
fixed